epub34/authoring/index.html

```diff
@@ -6215,6 +6215,22 @@ <h4>Introduction</h4>
 									<p>Some types of screens will render animations and transitions poorly (e.g., those
 										with high latency).</p>
 								</li>
+								<li>
+									<p>
+										Creators should be aware that a reading systems may override some aspects of 
+										the creator's style settings; creators may want to consult the reading systems' 
+										user agent style sheets, if publicly available. Furthermore, reading systems may
```

Comment:
Following Brady's previous comment, as we are in a NOTE section, it would be fair to replace it with a "will". Reading systems **will** definitely modify some rendering aspects as a result of user interaction.

Reply:
This could maybe be solved by adding more context: "EPUB creators should be aware than any reading system that allows users to modify the display will override some aspects..."